apps/teams-test-app/e2e-test-data/pages.json

```diff
@@ -22,9 +22,25 @@
       "type": "callResponse",
```

Comment:
We can use `version` or/and `hostSdkVersion` to exclude the original test case and then you will need to add two new cases.